docs/ruleset_functions/is_baseline_system_10.md

```diff
@@ -0,0 +1,40 @@
+# is_baseline_system_10  
+
+**Description:** Get either Sys-10 or Not_Sys_10 string output which indicates whether the HVAC system is ASHRAE 90.1 2019 Appendix G system 10 (Heating and Ventilation with electric heating).  
+
+**Inputs:**  
+- **B-RMR**: To evaluate if the hvac system is modeled as either Sys-10 or Not_Sys_10 in the B_RMR.   
+- **hvac_b.id**: The id of the hvac system to evaluate.  
+- **terminal_unit_id_list**: list of terminal unit IDs associated with the HVAC system to be evaluated. These are sent to this function from the master get_baseline_system_types function.
+- **zone_id_list**: list of zone IDs associated with the HVAC system to be evaluated. These are sent to this function from the master get_baseline_system_types function.
+
+**Returns:**  
+- **is_baseline_system_10**: The function returns either Sys-10 or Not_Sys_10 string output which indicates whether the HVAC system is ASHRAE 90.1 2019 Appendix G system 10 (Heating and Ventilation with electric heating).  
+ 
+**Function Call:** 
+1. is_hvac_sys_cooling_type_none()
+2. is_hvac_sys_fan_sys_CV()  
+3. is_hvac_sys_heating_type_elec_resistance()
+4. are_all_terminal_heat_sources_none_or_null()  
+5. are_all_terminal_cool_sources_none_or_null() 
+6. are_all_terminal_fans_null()  
+7. are_all_terminal_types_CAV()   
+8. does_each_zone_have_only_one_terminal()    
+9. does_hvac_system_serve_single_zone()  
+10. is_there_only_one_hvac_sys_heating_system ()  
+ 
+## Logic:    
+- Create an object associated with the hvac system: `hvac_b = hvac_b.id`  
+- Set is_baseline_system_10 = Not_Sys_10: `is_baseline_system_10 = "Not_Sys_10"`    
+- Check that there is one heating system, if there is then carry on: `if is_there_only_one_hvac_sys_heating_system(B_RMR, hvac_b.id) == TRUE:`   
+    - Check that there is no preheat system, if there is none then carry on: `if len(hvac_b.preheat_system) == Null or hvac_b.preheat_system[0].heating_system_type = "NONE" :`    
+        - Check if fansystem is constant volume, if yes then carry on: `if is_hvac_sys_fan_sys_CV(B_RMR, hvac_b.id) == TRUE:`  
+            - Check if the hvac system serves a single zone and that the zone only has one terminal unit: `if does_hvac_system_serve_single_zone(B_RMR, zone_id_list) == TRUE AND does_each_zone_have_only_one_terminal(B_RMR,zone_id_list) == TRUE:`     
+                - Check that the data elements associated with the terminal unit align with system 10: `if are_all_terminal_heat_sources_none_or_null(B_RMR,terminal_unit_id_list) == TRUE AND are_all_terminal_cool_sources_none_or_null(B_RMR,terminal_unit_id_list) == TRUE And are_all_terminal_fans_null(B_RMR,terminal_unit_id_list) == TRUE AND are_all_terminal_types_CAV(B_RMR,terminal_unit_id_list) == TRUE:`        
```

Comment:
I wonder if for single zone systems we need to check for CV vs VAV at both fan system level and terminal level, or only once, at the level where fans are defined. For example, here we would only check it at the fan system level (is_hvac_sys_fan_sys_CV(B_RMR, hvac_b.id) == TRUE) but not at the terminal level (are_all_terminal_types_CAV(B_RMR,terminal_unit_id_list) == TRUE)

Reply:
that would be consistent with how we verify FPFCU, i would be ok with that change. I'll hold off on marking this as ready to merge till we resolve this